knowledge_base/_Glossary.md

```diff
@@ -0,0 +1,250 @@
+# Glossary
+
+As of 240305, this starter glossary is in-progress and needs improvement. If you notice unclear wording or conceptual confusions, please open a PR tethered to the #4800 general documentation ticket!
+
+Glossary terms referenced in a given term's definition are designated with CAPITAL letters. Relevant models and related entries may also be referenced; see [Subscription](#subscription) for an example.
+
+## Contents
+
+- [Chain Concepts](#chain-concepts)
+  * [Chain Entity](#chain-entity)
+  * [Chain Node](#chain-node)
+  * [Delegate Contract](#delegate-contract)
+  * [Module](#module)
+- [Common App](#common-app)
+  * [Bridge](#bridge)
+  * [Comment](#comment)
+  * [Community](#community)
+  * [Core](#core)
+  * [Forum](#forum)
+  * [Group](#group)
+  * [Link](#link)
+  * [Member](#member)
+  * [Notification](#notification)
+  * [Page Scope](#page-scope)
+  * [Profile](#profile)
+  * [Reaction](#reaction)
+  * [Poll](#poll)
+  * [Role](#role)
+  * [Sessions](#sessions)
+  * [Sign-In](#sign-in)
+  * [Stake](#stake)
+  * [Subscription](#subscription)
+  * [Template](#template)
+  * [Thread](#thread)
+  * [Topic](#topic)
+  * [User](#user)
+  * [Wallet](#wallet)
+  * [Votes](#votes)
+- [Hexagonal Architecture](#hexagonal-architecture)
+  * [Port](#port)
+  * [Adapter](#adapter)
+- [Praxis](#praxis)
+  * [Blocked](#blocked)
+  * [Fast-Follow](#fast-follow)
+  * [Issues, Tickets, & Stories](#issues-tickets--stories)
+- [Change Log](#change-log)
+
+## Chain Concepts
+
+### Chain Base
+
+An architectural layer of blockchain atop which NETWORK protocols are built.
+
+### Chain Entity
+
+A Chain Entity is a backend object, stored in the ChainEntities table, which consists of chain event bundles and values received from a smart contract.
+
+### Chain Node
+
+An endpoint, referring to external RPC servers, used to fetch updates and query data from blockchains.
+
+### Delegate Contract
+
+A delegate contract is a smart contract that contains the functions, events, and all business logic for a proxy contract.
+
+### Module
+
+Some CHAINS use modules rather than contracts. Modules are used predominantly within the Cosmos and Substrate ecosystems.
+
+### Network
+
+An ecosystem of CHAIN NODES, or blockchain, built atop a CHAIN BASE.
+
+## Common App
+
+### Bridge
+
+The Product Team uses "bridge" and "bridging" language, in user-facing comms, to refer to implementation-agnostic, cross-platform integrations such as [Discobot](./Discobot.md) and Farcaster.
+
+### Comment
+
+A short Markdown text object responding to a THREAD, which can be REACTED to.
+
+See [models/comment.ts](../libs/model/src/models/comment.ts).
+
+### Community
+
+<!-- TODO: Review & update in light of new permissioning system; blocked pending meeting with Product. -->
+
+A community is a space on Common, joinable by PROFILES, that owns metadata and is defined by a set of ROLES and PERMISSIONS for participation.
```

Comment:
"Joinable" is not a clear distinction to make here, because Profiles join by way of Addresses, and also because joining is not an essential property of a community -- as part of #6959 we intend to clarify that Communities contain groups, which will support the function of "joining". ROLES are also not canonical objects -- we no longer have a Roles table, although we have admin/moderator/member statuses, they will be part of groups too. PERMISSIONS is also gone.

A community really is a unique namespace identifying a set of underlying applications on the Common app.